### ISSUE TITLE
[1.16.5] Add Spanish language support

### ISSUE THREAD

**Issue by FrannDzs** [Teneted/Tenet#1805]
Hi guys, how are you doing?, I hope well. I am using your Mohist software to join a server but I see that the spanish language is not translated, and I only see .json files corresponding to the french, russian and chinese language.  Is there any way you can implement the spanish language easily? I am a translator, so if you need help here I am. Greetings and I hope a prompt implementation of the language!

**Comment by Mgazul** [comment]
https://crowdin.com/project/mohist

**Comment by FrannDzs** [comment]
> https://crowdin.com/project/mohist

Hi, I checked what you recommended, but those are the server messages, I was looking to translate something like this:
`https://github.com/MohistMC/Mohist/tree/1.16.5/src/main/resources/assets/minecraft/lang`

Is it possible?, thank you in advance!
PD: One of the messages that is in English (I am a Spanish speaker) is when the player enters the server: `{player} join the server`. I was looking for it in the messages but I can't find it, I think it is in the path I marked. But it has some strange characters that I don't know how to skip since I can't find the language en_en in that folder and I suppose that they generate the language files with a script. Anything else I am listening to you!
